core/lib/workarea/release.rake

```diff
@@ -0,0 +1,105 @@
+require 'active_support/inflector'
+
+load 'workarea/changelog.rake'
+
+namespace :workarea do
+  desc "Release the next version of this gem"
+  task :release, [:private] do |task, args|
+    #
+    # Devise current gem version
+    #
+    #
+    version_file_path = Dir['**/version.rb'].reject { |p| p =~ /vendor/ }.first
+    gem_namespace =
+      version_file_path
+        .split('lib/').last
+        .split('/version').first
+        .classify
+
+    gem_name = gem_namespace.gsub(/::/, ' ')
+
+    current_version = if gem_namespace == 'Workarea'
```

Comment:
I'm kinda confused on all this stuff to get the version - I believe in the all gems if you just load the `whatever.gemspec`, it will `require` the right `lib/*/version.rb` file and then you can reference that. Am I missing something?

Reply:
I think I went this route because gems are likely to have one relevant `version.rb` file, but could have many `gemspec`s. Plus I needed the version file anyway, so grabbing the version out of there seemed reasonable.